app/src/main/java/com/pvp/app/service/ConfigurationImpl.kt

```diff
@@ -21,4 +23,6 @@ class ConfigurationImpl @Inject constructor() : Configuration {
         .toList()
 
     override val intervalDrinkReminder: Pair<Int, Int> = Pair(8, 22)
+
+    override val intervalUsernameLength: Pair<Int, Int> = Pair(3, 18)
```

Comment:
These could have stayed next to each other, since they are single-lined. If we want to separate configuration properties, then split all properties above too.